sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -173,6 +173,53 @@ export class BlobServiceClient extends StorageClient {
     });
   }
 
+  /**
+   * Iterates over containers under the specified account.
+   *
+   * @param {ServiceListContainersSegmentOptions} [options={}] Options to list containers(optional)
+   * @returns {AsyncIterableIterator<Models.ContainerItem>}
+   * @memberof BlobServiceClient
+   *
+   * @example
+   * for await (const container of blobServiceClient.listContainers()) {
```

Comment:
Just wondering what will happen if Aborter aborted iteration

Reply:
Example - 
```typescript
  // List containers - generator syntax
  let iter2 = await blobServiceClient.listContainers({ abortSignal: Aborter.timeout(X) });
  i = 1;
  let containerItem = await iter2.next();
  do {
    console.log(`Container ${i++}: ${containerItem.value.name}`);
    containerItem = await iter2.next();
  } while (containerItem.value);
```
[ About 6500 containers are present in the account ]

- When X = 50000 (large number), all the 6500 containers got listed(and their names are printed in the terminal)
- When X = 2000, only 5000 containers got listed and a message got printed saying `The request was aborted`.
![image](https://user-images.githubusercontent.com/10452642/58443794-23cd5b80-80a9-11e9-986f-7e1821edd059.png)
 [ X = 10000 yielded the same result ]
- When X < 1000, none of the containers got listed, `The request was aborted` is printed in the terminal.